docs/source/examples/demo_transformer_model_train_save_upload_to_openSearch.ipynb

```diff
@@ -163,19 +167,18 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "100%|██████████| 243/243 [00:00<00:00, 629923.28it/s]"
+      "100%|██████████| 243/243 [00:00<00:00, 148986.39it/s]"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "generated config file: at /home/ec2-user/.cache/huggingface/accelerate/default_config.yaml\n",
+      "generated config file: at /Users/dhrubo/.cache/huggingface/accelerate/default_config.yaml\n",
```

Comment:
Let's avoid using personal home folder

Reply:
I think these information are too much to show. I will add `isVerbose` field in this method to control the visibility of these messages in the next PR.